packages/app/src/AutocompleteInput.tsx

```diff
@@ -117,6 +143,37 @@ export default function AutocompleteInput({
             <div className="d-flex p-2 flex-wrap px-3">{aboveSuggestions}</div>
           )}
           <div>
+            {
+              // only show search history when: 1.no input, 2.has search type, 3.has history list
+              value.length === 0 &&
+                queryHistoryType &&
+                queryHistoryList.length > 0 && (
+                  <div className="border-top border-dark fs-8 py-2">
+                    <div className="text-muted fs-8 fw-bold me-1 px-3">
+                      Search History:
+                    </div>
+                    {queryHistoryList.map(({ value, label }, i) => {
+                      return (
+                        <div
+                          className={`py-2 px-3 text-muted ${
+                            selectedQueryHistoryIndex === i ? 'bg-hdx-dark' : ''
+                          }`}
+                          role="button"
+                          key={value}
+                          onMouseOver={() => {
+                            setSelectedQueryHistoryIndex(i);
+                          }}
+                          onClick={() => {
+                            onSelectSearchHistory(value);
+                          }}
+                        >
+                          <span className="me-1  flex-wrap">{label}</span>
+                        </div>
+                      );
+                    })}
+                  </div>
+                )
+            }
```

Comment:
I think this would flow better if search history was at the bottom so that the examples can pair with the syntax

![image](https://github.com/user-attachments/assets/58ea495e-6596-478f-8303-517aac0eaa3a)

Reply:
make sense, let me update it

Reply:
done
<img width="973" alt="image" src="https://github.com/user-attachments/assets/9dd4823a-266a-4ac2-84ee-7005d34629ae" />